Submissions/MichaelBoitmann.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Michael Boitmann",
+    "img": "https://avatars.githubusercontent.com/u/55775047?s=400&u=9125d841568874d0f4a54b158e9f17dd004b3921&v=4",
+    "email":"michael_boitmann@yahoo.com",
+    "links": {
+      "website": "",
+      "linkedin": "https://www.linkedin.com/in/michael-boitmann-85905a33/",
+      "github": "https://github.com/MichaelBoitmann"
+    },
+    "jobTitle": "Software Developer | ReactJS | Django |Python ",
```

Comment:
```suggestion
    "jobTitle": "Software Developer | ReactJS | Django | Python ",
```

Reply:
use removing cached file right after the cloning.
![image](https://github.com/zero-to-mastery/ZtM-Job-Board/assets/55775047/81d1c946-83d3-44b3-9000-b226641724f7)